mesonbuild/modules/hotdoc.py

```diff
@@ -18,30 +18,52 @@
 import os, subprocess
 import typing as T
 
-from mesonbuild import mesonlib
-from mesonbuild import mlog, build
-from mesonbuild.mesonlib import MesonException
-from . import ModuleReturnValue, ModuleInfo
-from . import ExtensionModule
+from . import ExtensionModule, ModuleReturnValue, ModuleInfo
+from .. import build, mesonlib, mlog
```

Comment:
perhaps drop build here?

Reply:
Why? It is used for other things besides the two that are directly imported. Same with mesonlib and File.

Reply:
You could just directly import what you want from build.

Reply:
That is technically feasible, I simply decided that I personally figured it would be frustrating to micromanage individual imports for every object used once.

So I only directly imported the very frequently used objects where line length benefitted from the conciseness.

It's very subjective, I agree. Do you strongly prefer importing them all?

Reply:
Nah I don't care. As soon as CI passes looks good to merge